File: Domain/Entities/VehicleAnormality.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain.Entities
{
public class VehicleAnormality: BaseEntity
{
    public int Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public DateTime EntryDate { get; set; }
    public ICollection<VehicleAnormalityDetail>? VehicleAnormalityDetails { get; set; }
}
}

File: autotaller-manager/src/app/modules/admin/models/vehicle-anormality.ts
import { VehicleAnormalityDetail } from "./vehicle-anormality-detail";

export interface VehicleAnormality {
  id: number;
  name: string;
  entryDate: string; 
    
  createdAt?: Date;
  updatedAt?: Date;
  vehicleAnormalityDetails?: VehicleAnormalityDetail[];
}
